libs/clients/cbr/client.go

```diff
@@ -86,6 +123,23 @@ type CredentialsIssueResponse struct {
 	SignedTokens []string `json:"signed_tokens"`
 }
 
+// GetIssuerV2 - get a v2 issuer from CB
+func (c *HTTPClient) GetIssuerV2(ctx context.Context, issuer string, cohort int16) (*IssuerResponse, error) {
+	payload := struct {
+		Cohort int16 `json:"cohort"`
+	}{cohort}
+
+	req, err := c.client.NewRequest(ctx, http.MethodGet, "v2/issuer/"+issuer, &payload, nil)
```

Comment:
I don't know how I missed this in the challenge bypass review but please let's not pass cohort in the body on a GET request, let's use a query parameter...

Reply:
tracking here: https://github.com/brave-intl/challenge-bypass-server/issues/290

Reply:
this pr removes the passing of a body in a get request for v3 apis. https://github.com/brave-intl/challenge-bypass-server/pull/292